livekit-android-sdk/src/main/java/io/livekit/android/room/participant/LocalParticipant.kt

```diff
@@ -850,6 +850,13 @@ abstract class BaseVideoTrackPublishOptions {
      */
     abstract val scalabilityMode: String?
 
+    /**
+     * Multi-codec Simulcast
+     *
+     * Codecs such as VP9 and AV1 are not supported by all clients. When backupCodec is
+     * set, when an incompatible client attempts to subscribe to the track, LiveKit
+     * will automatically publish a secondary track encoded with the backup codec.
+     */
     abstract val backupCodec: BackupVideoCodec?
```

Comment:
is this default enabled on Android? or something you'd need to set?

Reply:
Always on for svc codecs atm, will need to address being able to turn it off for svc codecs. Otherwise empty for other codecs.

Reply:
cool, yeah let's discuss default behavior, ideally it's consistent across SDKs